packages/curve-ui-kit/src/utils/searchText.ts

```diff
@@ -1,5 +1,7 @@
 import { get, isEqualWith, uniqWith } from 'lodash'
 import Fuse, { FuseResult } from 'fuse.js'
+import type { EndsWith } from 'main/src/dex/components/ComboBoxSelectToken/types'
```

Comment:
Not blocking the PR for this but importing app related stuff into curve-ui-kit really rustles my jimmies

Reply:
this is bad yea, we should add this to the eslint rules

Reply:
#666